app/routers/event.py

```diff
@@ -104,3 +105,36 @@ def sort_by_date(events: List[Event]) -> List[Event]:
 
     temp = events.copy()
     return sorted(temp, key=attrgetter('start'))
+
+
+def check_date_validation(start_time, end_time) -> bool:
+    """Check if the start_date is smaller then the end_time"""
+
+    try:
+        return start_time < end_time
+    except TypeError:
+        return False
+
+
+def add_new_event(values: dict, db) -> Optional[Event]:
+    """Get User values and the DB Session insert the values
+    to the DB and refresh it exception in case that the keys
+    in the dict is not match to the fields in the DB
+    return the Event Class item"""
+
+    if check_date_validation(values['start'], values['end']):
```

Comment:
Prefer `if not ...` to [return early](https://medium.com/swlh/return-early-pattern-3d18a41bba8)